### PR TITLE
Messaging extension anuj

### PR DESCRIPTION
**Pull request by anbah34**
Called out functionality that is not supported on mobile yet on this page

### CODE REVIEW

File: msteams-platform/messaging-extensions/design/messaging-extension-design.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ ms.topic: conceptual
 # Designing your Microsoft Teams messaging extension
 
 Messaging extensions are shortcuts for inserting app content or acting on a message without navigating away from the conversation.
-
+Hi anuj here
```
Comment (KirtiPereira) [inline msteams-platform/messaging-extensions/design/messaging-extension-design.md:12]: Please remove this.

**Review by KirtiPereira — CHANGES_REQUESTED** [review]
@anbah34 - Please incorporate the comment and resolve the build warnings.

**Review by surbhigupta12 — APPROVED** [review]
Approving the PR

**Review by KirtiPereira — APPROVED** [review]
approved

**Pull request merged**